java/org/contikios/cooja/plugins/ScriptRunner.java

```diff
@@ -150,6 +150,25 @@ public ScriptRunner(Simulation simulation, Cooja gui) {
     logTextArea.setEditable(true);
     logTextArea.setCursor(null);
 
+    final var NEWLINE = System.getProperty("line.separator");
+    var open = new JMenuItem("Open...");
+    open.addActionListener(l -> {
+      var f = showFileChooser(true);
+      if (f == null) {
+        return;
+      }
+      try (var reader = Files.newBufferedReader(f.toPath(), UTF_8)) {
+        var sb = new StringBuilder(4096);
+        String line;
+        while ((line = reader.readLine()) != null) {
+          sb.append(line).append(NEWLINE);
+        }
+        codeEditor.setText(sb.toString());
```

Comment:
This is a nice addition to being able to read and save scripts directly from the plugin. To simplify the code, the file could be read using `StringUtils.loadFromFile()`as is already done in `activateScript()`.

Reply:
Fixed.

Reply:
Thanks!